### ISSUE TITLE
.net framework, can't validate XML documents with an XML schema having targetNamespace

### ISSUE THREAD

**Issue by mukulga** [dotnet/runtime#67283]
### Description

The XML schema (XSD) validation example, mentioned at https://docs.microsoft.com/en-us/dotnet/standard/data/xml/xml-schema-xsd-validation-with-xmlschemaset, doesn't validate properly due to the presence of targetNamespace attribute within the XML schema document.

### Reproduction Steps

Please use the complete XML schema validation example, described at https://docs.microsoft.com/en-us/dotnet/standard/data/xml/xml-schema-xsd-validation-with-xmlschemaset. If within the XML instance document example, posted on the mentioned link, we change the value of "xmlns" at the top (for e.g, if we change that to http://www.contoso.com/books1), the XSD validation doesn't generate any error.

### Expected behavior

The XSD validation, should produce an error like, "XML element not declared", or "element not declared".

### Actual behavior

The XSD validation, doesn't produce any error message.

### Configuration

.NET version : 6.0, 5.0

OS : Windows 11

CPU architecture : x64

**Comment by krwq** [comment]
@mukulga good catch - example probably should have `booksSettings.ValidationFlags = XmlSchemaValidationFlags.ReportValidationWarnings;`

Missing schema for unknown namespace is considered a warning not an error and since warnings are not reported you're seeing empty output.

note that this is not specific to .NET Framework also repros on .NET 7 and presumably all version before.

**Comment by mukulga** [comment]
> @mukulga good catch - example probably should have `booksSettings.ValidationFlags = XmlSchemaValidationFlags.ReportValidationWarnings;`
> 
> Missing schema for unknown namespace is considered a warning not an error and since warnings are not reported you're seeing empty output.
> 
> note that this is not specific to .NET Framework also repros on .NET 7 and presumably all version before.

Thanks, for your reply.

I'm happy with your suggestion to have, booksSettings.ValidationFlags = XmlSchemaValidationFlags.ReportValidationWarnings; That gives me all the other information I need, for the cited XML Schema validation example.
